### ISSUE TITLE
[BUG] - GEN LOAD HI When On APU Power

### ISSUE THREAD

**Issue by GR1MMly** [invernyx/md11-bugs#213]
### Describe the bug

When powering on APU and connecting via electrical panel, lvl 2 GEN LOAD HI is persistent despite no high loads

### Steps to reproduce

Bat On, Ext Pwr On
Start APU
Once APU is on speed, level 2 GEN LOAD HI Alert will be displayed

### Expected behaviour

Alert should not be displayed unless an abnormal generator load is present.

### Simulator

Microsoft Flight Simulator 2020 (MSFS)

### Crash

No

### Phase of flight

Pre-flight

### Altitude

0

### Agreement

- [X] Yes

**Comment by FreakyD777** [comment]
I cannot recreate this issue

**Comment by GR1MMly** [comment]
> I cannot recreate this issue as you describe above. However, I can recreate this with the following procedure BAT ON GROUND PWR ON START APU (either on APU panel or ELEC panel) IRS 1 to NAV IRS 2 to NAV message will post at this point

You are correct, I failed to include in the original that the IRS were in NAV. Fixed

**Comment by turbofandude** [comment]
Resolved in 0.53+
